Engines/Wine/QuickScript/Zip Script/script.js

```diff
@@ -15,6 +17,11 @@ module.default = class ZipScript extends QuickScript {
         return this;
     }
 
+    setupPathInsidePrefix(setupPath) {
```

Comment:
Please add comments about:
- how `ZipScript` behaves depending on whether a setup path is specified or not
- what the argument means and what type it is
- that the usage of the method is purely optional